src/Shiny.Notifications/Platforms/Android/NotificationManager.cs

```diff
@@ -217,14 +228,32 @@ protected virtual PendingIntent GetLaunchPendingIntent(Notification notification
         }
 
 
+        protected virtual int? GetColor(string colorResourceName)
+        {
+            if (colorResourceName.IsEmpty())
+                return null;
+
+            var colorResourceId = this.context.GetColorByName(colorResourceName);
+            if (colorResourceId <= 0)
+            {
+                Log.Write(NotificationLogCategory.Notifications, $"Color ResourceId for {colorResourceName} not found");
```

Comment:
Always throw on bad args from the user - otherwise they tend to file bugs saying it isn't working, where in actual fact, it isn't finding the resource because they haven't set it up